src/app/main/api-gest-console/modules/aig-standard/components/cpv-detail-page/cpv-detail-page.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
fix this page take ispiration from this file: [https://github.com/EneaSys/aig_console/blob/develop/src/app/main/api-gest-console/modules/aig-generic/components/eopoo-type-detail-page/eopoo-type-detail-page.component.ts](url)